gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/task/EhrExtractMessageHandlerTest.java

```diff
@@ -480,6 +490,54 @@ public void When_HandleSingleMessageWithSkeletonAttachmentAndValidDataIsCalled_E
         verify(skeletonProcessingService, times(1)).updateInboundMessageWithSkeleton(any(), any(), any());
     }
 
+    @Test
+    public void When_HandleSingleMessageWithAFilenameDescription_Expect_PatientAttachmentLogEntryGenerated()
+            throws JsonProcessingException,
+            JAXBException,
+            InlineAttachmentProcessingException,
+            BundleMappingException,
+            AttachmentNotFoundException,
+            ParseException,
+            SAXException, TransformerException, UnsupportedFileTypeException {
+
+        InboundMessage inboundMessage = new InboundMessage();
+        var attachment = new InboundMessage.Attachment("text/xml",
+                "true", "test.txt", "abcdefghi");
+
+        var attachmentList = new ArrayList<InboundMessage.Attachment>();
+        attachmentList.add(attachment);
+        inboundMessage.setAttachments(attachmentList);
+
+        inboundMessage.setPayload(readInboundSingleMessagePayloadFromFile());
+        inboundMessage.setEbXML(readInboundSingleMessageEbXmlFromFile());
+
+        prepareMigrationRequestAndMigrationStatusMocks();
+
+        when(xPathService.getNodeValue(any(), any())).thenReturn("MESSAGE-ID");
+        when(attachmentReferenceUpdaterService
+                .updateReferenceToAttachment(
+                        inboundMessage.getAttachments(), CONVERSATION_ID, inboundMessage.getPayload()
+                )).thenReturn(inboundMessage.getPayload());
+
+        ehrExtractMessageHandler.handleMessage(inboundMessage, CONVERSATION_ID);
+
+        verify(patientAttachmentLogService, times(1)).addAttachmentLog(patientAttachmentLogCaptor.capture());
+        PatientAttachmentLog log = patientAttachmentLogCaptor.getValue();
+        assertEquals("MESSAGE-ID", log.getMid());
+        assertEquals("test.txt", log.getFilename());
+        assertNull(log.getParentMid());
+        assertEquals(MIGRATION_REQUEST_ID, log.getPatientMigrationReqId());
+        assertEquals(attachment.getContentType(), log.getContentType());
+        assertFalse(log.getCompressed());
+        assertFalse(log.getLargeAttachment());
+        assertTrue(log.getOriginalBase64()); // _Should_ this be true?
```

Comment:
Is this correct?

Reply:
I _don't think_ this field is used for anything, but probably safer to make it false?